packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -243,9 +243,10 @@ class _CircularProgressIndicatorPainter extends CustomPainter {
     if (value == null) // Indeterminate
       paint.strokeCap = StrokeCap.square;
 
-    Path path = new Path()
-      ..arcTo(Point.origin & size, arcStart, arcSweep, false);
-    canvas.drawPath(path, paint);
+    // Path path = new Path()
+    //   ..arcTo(Point.origin & size, arcStart, arcSweep, false);
+    // canvas.drawPath(path, paint);
```

Comment:
@Hixie Definitely not! This PR is just groundwork until I roll the engine. (which should happen soon)